examples/gsd/_03_dmo_evaluation_on_wb_level.py

```diff
@@ -0,0 +1,251 @@
+#    where `<metric>` is the name of the DMO to evaluate, `<origin>` is the column from which data should be utilized
+#    (for this example, it would be either `detected`, `reference`, or one of the error columns)
+#    and `<aggregation>` is the function or the list of functions to apply.
+#    The output dataframe will have a multilevel column with `metric` as the first level and
+#    `origin` as the second level.
+#    A valid aggregations list for all of our DMOs would consequently look like this:
+#
+
+aggregations_simple = [*(((m, o), ["mean", "std"]) for m in metrics for o in ["detected", "reference", "error"])]
+print(aggregations_simple)
+
+# %%
+#
+# 2. As a named tuple of Type `CustomOperationTuple` taking three values: `identifier`, `function`, and `column_name`.
+#   `identifier` is a valid loc identifier selecting one or more columns from the dataframe,
+#   `function` is the (custom) aggregation function or list of functions to apply,
+#   and `column_name` is the name of the resulting column in the output dataframe
+#   (single-level column if `column_name` is a string, multi-level column if `column_name` is a tuple).
+#   This allows for more complex aggregations that require multiple columns as input, for example, the intraclass
+#   correlation coefficient (ICC) for the DMOs.
+#   A valid aggregation list for calculating the ICC of all DMOs would look like this:
+#
+
+aggregations_custom = [CustomOperationTuple(identifier=m, function=icc, column_name=(m, "all")) for m in metrics]
+print(aggregations_custom)
+
+# %%
+# Within one aggregation list, both types of aggregations can be combined
+# as long as the resulting output dataframes can be concatenated, i.e. have the same number of column levels.
+# Then, the :func:`~mobgap.gsd.evaluation.apply_aggregations` function can be called.
+# For better readability, the index of the aggregation result dataframe is named and sorted.
+from mobgap.gsd.evaluation import apply_aggregations
+
+aggregations = aggregations_simple + aggregations_custom
+agg_results = apply_aggregations(gs_matches_with_errors, aggregations)
+agg_results.index.names = ["aggregation", "metric", "origin"]
+agg_results = agg_results.reorder_levels(["metric", "origin", "aggregation"]).sort_index(level=0)
+print(agg_results)
+
+# %%
+# If you simply want to apply a standard set of aggregations to the error metrics, you can use the
+# :func:`~mobgap.gsd.evaluation.get_default_aggregations` function, resulting in the following list:
+from mobgap.gsd.evaluation import get_default_aggregations
+
+aggregations_default = get_default_aggregations()
+print(aggregations_default)
+
+# %%
+# This list of standard aggregations can then also be passed to the :func:`~mobgap.gsd.evaluation.apply_aggregations`
+# function.
+default_agg_results = apply_aggregations(gs_matches_with_errors, aggregations_default)
+default_agg_results.index.names = ["aggregation", "metric", "origin"]
```

Comment:
Instead of `index.names` you canuse `.rename_axis(index=...)`. That is easier to read and chainable